spec/system/jobseekers_can_view_job_facet_landing_pages_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe "Jobseekers can view job facet landing pages" do
+  before do
+    store = MockRedis.new
+    store.set(:job_roles, { teacher: 1 }.to_json)
+    store.set(:subjects, { Bengali: 5 }.to_json)
+    store.set(:cities, { London: 10 }.to_json)
+    store.set(:counties, { Devon: 15 }.to_json)
+    vacancy_facets = VacancyFacets.new(store: store)
+    allow(VacancyFacets).to receive(:new).and_return(vacancy_facets)
+    home_page.load
+  end
+
+  describe "job roles" do
+    it "jobseekers can view landing pages" do
```

Comment:
does it read well? maybe we should read it starting with something like

`it checks the job roles filter on results page`